Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -773,7 +773,7 @@ def test_pull
     end
 
     assert_match "Testing URLs require `--bottle`!",
-      cmd_fail("pull", "http://bot.brew.sh/job/Homebrew\%20Testing/1028/")
+      cmd_fail("pull", "https://bot.brew.sh/job/Homebrew\%20Testing/1028/")
```

Comment:
It's just a placeholder, it could point to anything in the testing section, even something that doesn't exist:

```
~> brew pull https://bot.brew.sh/job/Homebrew\%20Testing/9999/
Error: Testing URLs require `--bottle`!
```
I'm only really updating it to `https` for the sake of consistency.